cypress/e2e/map.cy.ts

```diff
@@ -1,25 +1,25 @@
-import driver from "./driver";
+import MaputnikDriver from "./driver";
 
 describe("map", () => {
-    describe("zoom level", () => {
-      beforeEach(() => {
-        driver.beforeEach();
-      });
-      it("via url", () => {
-        var zoomLevel = 12.37;
-        driver.setStyle("geojson", zoomLevel);
-        driver.isDisplayedInViewport(".maplibregl-ctrl-zoom");
-        // HM TODO
-        //driver.getText(".maplibregl-ctrl-zoom") === "Zoom "+(zoomLevel);
-      })
-      it("via map controls", () => {
-        var zoomLevel = 12.37;
-        driver.setStyle("geojson", zoomLevel);
-  
-        driver.click(".maplibregl-ctrl-zoom-in");
-        driver.isDisplayedInViewport(".maplibregl-ctrl-zoom");
-        // HM TODO
-        //driver.getText(".maplibregl-ctrl-zoom") === "Zoom "+(zoomLevel + 1);
-      })
-    })
-  })
+  let { beforeAndAfter, given, when, get, should } = new MaputnikDriver();
+  beforeAndAfter();
+  describe("zoom level", () => {
+    it("via url", () => {
+      var zoomLevel = 12.37;
+      when.setStyle("geojson", zoomLevel);
+      should.beVisible("maplibre:ctrl-zoom");
+      // HM TODO
+      //driver.getText(".maplibregl-ctrl-zoom") === "Zoom "+(zoomLevel);
```

Comment:
Can the helper help here? I didn't know how to write it and left it here by mistake...
This tests makes sure the zoom's text in the main window changes when you click the zoom button.
Top right corner:
![image](https://github.com/maputnik/editor/assets/3269297/b7066c57-c2c1-4049-ac97-e164f4a5f69e)